sbin/common/config_init.sh

```diff
@@ -271,8 +271,14 @@ function parseConfigurationArguments() {
         "--destination" | "-d" )
         BUILD_CONFIG[TARGET_DIR]="$1"; shift;;
 
-        "--docker" | "-D" )
-        BUILD_CONFIG[USE_DOCKER]="true";;
+        "-D" )
+        if [ which podman ] ; then BUILD_CONFIG[USE_DOCKER]="podman" ; else BUILD_CONFIG[USE_DOCKER]="docker" ; fi;;
```

Comment:
This was picked up by the linter (which will need some more things addressing):
```suggestion
        if which podman > /dev/null ; then BUILD_CONFIG[USE_DOCKER]="podman" ; else BUILD_CONFIG[USE_DOCKER]="docker" ; fi;;
```

Reply:
Yah, I know. I was not yet adressing. them. Will address once (an if at all :) ) this will be ever moved to final working state. Will commit the suggestion anyway.

Reply:
ty!

Reply:
Yeah most of them are just the linter generally complaining about syntax, but since this one was a definite functional problem I figured it was best to mention it explicitly :-)

Reply:
yy.  face-palmed myself when I saw it. Thanx a lot.